src/methods/checkLang.ts

```diff
@@ -6,23 +6,25 @@ export declare namespace checkLang {
 
 export const checkLang = (): checkLang.langCode => {
   const mylang = document.getElementById("mylang")
+  if (!mylang) return "ja"
 
-  if (mylang) {
-    if (mylang.className.includes("mylang-ja")) {
+  return checkLangByDOMElement(mylang) ?? "ja"
+}
+
+const checkLangByDOMElement = (
+  mylang: HTMLElement
+): checkLang.langCode | undefined => {
+  switch (mylang.className) {
```

Comment:
This could easily be broken when some new class names are added (by the service). I believe we shoud do:

```typescript
  if (mylang.classList.contains("mylang-ja")) return "ja"
  if (mylang.classList.contains("mylang-en")) return "en"
```